MathLib/mechanics_invariants.h

```diff
@@ -33,7 +33,7 @@ struct Invariants final
     static Eigen::Matrix<double, KelvinVectorSize, KelvinVectorSize> const
         P_sph;
     /// Kelvin mapping of 2nd order identity tensor.
-    static Eigen::Matrix<double, KelvinVectorSize, 1> const identity2;
+    static Eigen::Matrix<double, KelvinVectorSize, 1> const Identity2;
```

Comment:
Changed P_dev/sph to deviatoric_projection and spherical_projection. Going back to identity2. Now all identifier are "style-guided" but J2 and J3, which are difficult to name in short manner and are common used under this name in formulas.